src/scripts/main.js

```diff
@@ -1,3 +1,238 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+const form = document.createElement('form');
+
+form.className = 'new-employee-form';
+
+const fields = [
+  {
+    label: 'Name: ',
+    name: 'name',
+    type: 'text',
+    qa: 'name',
+  },
+  {
+    label: 'Position: ',
+    name: 'position',
+    type: 'text',
+    qa: 'postion',
```

Comment:
There's a typo in the 'qa' attribute for the 'Position' field. It should be 'qa: "position"' instead of 'qa: "postion"'. This will cause issues when trying to query the input element by this attribute.